pkg/features/features.go

```diff
@@ -54,6 +58,7 @@ var (
 		CustomizedClusterResourceModeling: {Default: true, PreRelease: featuregate.Beta},
 		PolicyPreemption:                  {Default: false, PreRelease: featuregate.Alpha},
 		MultiClusterService:               {Default: false, PreRelease: featuregate.Alpha},
+		PolicyLazyReconcile:               {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Why is there only one global switch? Will you also introduce related fields in PP next?

What I mean is, can the system have both an immediately effective PP and a `Lazy Reconcile` PP at the same time?

Reply:
Because if there's only one of two options, then assuming all future defaults are `Lazy Reconcile`, most users would refuse to upgrade due to their lack of experience with `Lazy Reconcile`.

Reply:
+1
`lazy` should not be the default behavior.

Reply:
Lazy is not the default behavior as `PolicyLazyReconcile{Default: false}`

What chaunceyjiang mean is that it is better to add a filed to Policy to control the the effect strategy.

Reply:
What I mean is: It would be best if these two behaviors could coexist in the karmada.

Reply:
I still have one question I don't fully understand, which is: if a resource has not changed but the PP has changed, then this resource will continue to use the old PP. If a rescheduling happens at this time, should the scheduler use the old PP or the new PP? If it uses the new PP, then that behavior would be somewhat inconsistent with our expected behavior. 
Our expectation is that the new PP would only be applied if the resource itself has changed.